src/schema/users.ts

```diff
@@ -1873,6 +1879,35 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         true,
       );
     },
+    plusGifterUser: async (_, __, ctx: AuthContext, info): Promise<GQLUser> => {
+      const { subscriptionFlags } = await ctx.con
+        .getRepository(User)
```

Comment:
We can also query this from read replica

Reply:
Updated ✔️